source/mainnet/net/guides/change-identity-name.rst

```diff
@@ -7,13 +7,16 @@
 Change identity name
 ====================
 
-#. Go to the |morepage| page.
+.. dropdown:: |mw-gen2|
 
-#. Tap **Your identity cards** page. Tap the identity you want to edit.
+    #. Tap |wallet-settings|. Select **Your identity cards**.
 
-#. On the identity card tap |edit| next to the identity name.
+    #. Tap the identity you want to edit.
+
+    #. On the identity card tap |edit| next to the identity name.
+
+    #. Change the name and tap **Save**.
 
-#. Change the name and tap **Save**.
 
 .. |edit|    image:: ../images/edit.png
```

Comment:
I'm not sure if it is important, but isn't the background for this icon white in the MW? 

![image](https://user-images.githubusercontent.com/76045015/204250018-9bddddc2-088a-4ddf-97b8-c5200ab5d98d.png)

Reply:
It would be nice if Edit looked the same across all wallets. :-)